packages/flutter_tools/lib/src/build_system/targets/windows.dart

```diff
@@ -105,16 +105,13 @@ class DebugBundleWindowsAssets extends Target {
 
   @override
   List<Source> get inputs => const <Source>[
-    Source.pattern('{BUILD_DIR}/app.dill'),
     Source.pattern('{FLUTTER_ROOT}/packages/flutter_tools/lib/src/build_system/targets/windows.dart'),
     Source.pattern('{PROJECT_DIR}/pubspec.yaml'),
     ...IconTreeShaker.inputs,
   ];
 
   @override
-  List<Source> get outputs => const <Source>[
-    Source.pattern('{OUTPUT_DIR}/flutter_assets/kernel_blob.bin'),
-  ];
+  List<Source> get outputs => const <Source>[];
```

Comment:
Since this class is abstract anyway I would leave out the empty outputs

Reply:
Moved down to the subclasses.